### ISSUE TITLE
Mouseover tool tips not accessible to keyboard-only or other AT users – they should be

### ISSUE THREAD

**Issue by alisan16** [department-of-veterans-affairs/caseflow#12141]
https://www.sarasoueidan.com/blog/accessible-tooltips/

Information regarding a hearing is displayed with Mouse Overs. This information cannot be viewed with a keyboard or read by assistive technology programs. (New: 08/21/2019)
